spec.html

```diff
@@ -30395,7 +30395,7 @@ <h1>Runtime Semantics: UnicodeMatchProperty ( _p_ )</h1>
           <p>Implementations must only recognize the property aliases listed in <emu-xref href="#table-nonbinary-unicode-properties"></emu-xref> and <emu-xref href="#table-binary-unicode-properties"></emu-xref>.</p>
           <p>Implementations must only recognize the property value aliases and canonical property value names listed in <emu-xref href="#table-unicode-general-category-values"></emu-xref> and <emu-xref href="#table-unicode-script-values"></emu-xref>.</p>
           <emu-note>
-            <p>For example, `Script_Extensions` (property name) and `scx` (property alias) are valid, but `script_extensions` or `Scx` aren’t.</p>
+            <p>For example, `Script_Extensions` (property name) and `scx` (property alias) are valid, but `script_extensions` or `Scx` aren't.</p>
```

Comment:
typographically, `‘` is correct here (and `'` isn't); why these changes?

Reply:
consistency with the rest of the spec. (scan for /[a-z]'[a-z]/)

Reply:
Gotcha, that's unfortunate.